core/controllers/ingest.py

```diff
@@ -214,3 +304,46 @@ def save_submission_file(excel_file, submission_id):
     submission.submission_file = excel_file.stream.read()
     db.session.add(submission)
     db.session.commit()
+
+
+def modify_schema(schema):
+    """
+    Modifies the schema by removing specified tables and adding "Submission ID" as the first column to specified tables.
+
+    :param schema: The original schema dictionary.
+    :return: The updated schema dictionary.
+    """
```

Comment:
I know I suggested this way of doing it but I've thought on it and I actually think it's more maintainable and less error prone to have 2 separate verbose validation schemas.

- The main reason is that if we make changes to the validation schema in the current setup - it may have unintended affects on R1.
- It'll also be clearer what the validation logic is for each round if we give round 1 it's own verbose schema
- Member

Starting with the current schema and just making these changes shouldn't take very long so probably worth doing IMO

Reply:
I agree, and have updated the code to follow this approach.

```
schema = (
        current_app.config["ROUND_ONE_TF_VALIDATION_SCHEMA"]
        if source_type == "tf_round_one"
        else current_app.config["VALIDATION_SCHEMA"]
    )
```

This is going to quickly become messy with more rounds, but we can't define these in a dictionary outside of the function like for ETL_Pipelines as current app requires a context in which to operate. 

I am happy to refactor this - and other instances of if-statements to check r1 v. r3 - in another PR, but it could be somewhat time-consuming and is arguably beyond the scope of just loading in R1 if we're aiming to get this in today.

Reply:
Completely agree - I've been holding off commenting on things because we'll need to go back and think about this and refactor a lot of this code so it's more extendable. Will be good to get clear direction on the expected functionality going forwards and refactor accordingly.